.github/workflows/check_version.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           SNAP_VERSION=$(yq .version snap/snapcraft.yaml)
           APT_VERSION=$(apt info mysql-server | grep Version | cut -c10-)
-          APT_VERSION=${APT_VERSION::-15}
+          APT_VERSION=${APT_VERSION::-17}
```

Comment:
I think we need to add some love here:
1) we are installing debian package `mysql-server-8.0` but checking version of metapackage `mysql-server`.
2) `apt info` shows version in "apt metadata" and not the version of "installed" debian package.
2) "apt" produces warnings on STDERR: `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`
3) cutting last 17 symbols is an interesting approach... maybe this is nicer:
```
APT_VERSION=$(dpkg-query --show --showformat="\${source:Upstream-Version}" "mysql-server-8.0")
```

Reply:
Discussed offline: `dpkg-query` only works for installed snaps. Updated to use `apt info` on `mysql-server-8.0` (instead of `mysql-server`) in [66559e9](https://github.com/canonical/charmed-mysql-snap/pull/36/commits/66559e9e60c069b092c7ba4b2eaba742f387f70e)

Reply:
Use `apt-cache show`'s latest version entry in the version check (to address apt's unstable cli interface)